site/content/en/blog/releases/1.17.0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "1.17.0 - Kubernetes 1.20, deprecated required/preferred, add sync property to FleetAutoScaler"
+linkTitle: "1.17.0"
+date: "2021-08-31"
+---
+
+
+This is the 1.17.0 release of Agones.
```

Comment:
WDYT about adding some text here about the advanced allocation stuff mark has been doing?